include/openssl/e_os2.h

```diff
@@ -276,6 +276,18 @@ typedef unsigned __int64 uint64_t;
 #  endif
 # endif
 
+/* Format specifier for printing size_t */
+# if (defined(__STDC_VERSION__) && __STDC_VERSION__ >= 199901L) \
+      && !defined(__VMS)
+#  define OSSLzu  "zu"
+# else
+#  ifdef SIXTY_FOUR_BIT_LONG
+#   define OSSLzu PRIu64
```

Comment:
I think that both actually have a problem. In theory size_t could be an int and not a long, and so you'll get a warning that it's expecting a long and gets an int. (It could even be a short, but that gets promoted to an int anyway.) So we might need both "u" and "lu". I guess we should just wait until someone actually complains about this.

But the PRIu64 is probably more problematic, because we're on platform that doesn't have C99. So we could at least check that PRIu64 is a define or not. We should probably use "%llu" here, but I think I at least remember some platforms that wanted "%lu" even if long and long long are the same size.